Fail to verify with maven

[INFO] Results:                                                                                                        
[INFO]                                                                                                                  
[ERROR] Failures:                                                                                                       
[ERROR]   GT4500Test.fireTorpedo_All_Success:37 expected: <true> but was: <false>                                       
[INFO]                                                                                                                  
[ERROR] Tests run: 2, Failures: 1, Errors: 0, Skipped: 0                                                                                                                                                          
 [INFO] ------------------------------------------------------------------------                                        
 [INFO] BUILD FAILURE                                                                                                   
 [INFO] ------------------------------------------------------------------------                                         
[INFO] Total time:  13.909 s                                                                                            
[INFO] Finished at: 2023-09-27T09:08:06+02:00                                                                           
[INFO] ------------------------------------------------------------------------                                         
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.2:test (default-test) on project hu.bme.mit.spaceship: There are test failures.

--- a/pom.xml
+++ b/pom.xml
@@ -13,7 +13,7 @@
 		<dependency>
 			<groupId>org.junit.jupiter</groupId>
 			<artifactId>junit-jupiter</artifactId>
-			<version>5.8.2</version>
+			<version>5.10.0</version>
 			<scope>test</scope>
 		</dependency>
 		<dependency>

--- a/src/main/java/hu/bme/mit/spaceship/GT4500.java
+++ b/src/main/java/hu/bme/mit/spaceship/GT4500.java
@@ -78,7 +78,10 @@ public class GT4500 implements SpaceShip {
 
       case ALL:
         // try to fire both of the torpedo stores
-        //TODO implement feature
+        if((! primaryTorpedoStore.isEmpty()) && (! secondaryTorpedoStore.isEmpty())){
+          firingSuccess = primaryTorpedoStore.fire(1);
+          firingSuccess = secondaryTorpedoStore.fire(1);
+        }
 
         break;
     }

